src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,66 @@
-export const PeopleFilters = () => {
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { FilterSex } from '../types/Enums';
+
+type Props = {
+  activeSex: string | null;
+  filterQuery: string | null;
+  activeCenturies: string[];
+};
+
+export const PeopleFilters = ({
+  activeSex,
+  filterQuery,
+  activeCenturies,
+}: Props) => {
+  const [filterInputValue, setFilterInputValue] = useState(filterQuery || '');
```

Comment:
this is not a filtered value, but a search value
```suggestion
  const [searchValue, setSearchValue] = useState(filterQuery || '');
```